api/src/main/java/org/openmrs/api/impl/EncounterServiceImpl.java

```diff
@@ -167,7 +167,7 @@ public Encounter saveEncounter(Encounter encounter) throws APIException {
 
 		// save the new orderGroups
 		for (OrderGroup orderGroup : encounter.getOrderGroups()) {
-			Context.getOrderService().saveOrderGroup(orderGroup);
+			Context.getOrderService().saveOrderGroup(orderGroup,null);
```

Comment:
Why is this change necessary?  Can't we continue to call the existing method?

Reply:
yes, this change is not necessary, let me revert